File: Unity/UnityKakaotalkAPI/Scripts/Models/Account.cs
using System;

namespace Kakaotalk.Model
{
    [Serializable]
    public class Account {
        public AgeRange? age_range;
        public bool? age_range_needs_agreement;
        public string birthday;
        public bool? birthday_needs_agreement;
        public string birthyear;
        public bool? birthyear_needs_agreement;
        public string ci;
        public DateTime? ci_authenticated_at;
        public bool? ci_needs_agreement;
        public string email;
        public bool? email_needs_agreement;
        public Gender? gender;
        public bool? gender_needs_agreement;
        public bool? is_email_valid;
        public bool? is_email_verified;
        public string legal_birth_date;
        public bool? legal_birth_date_needs_agreement;
        public Gender? legal_gender;
        public bool? legal_gender_needs_agreement;
        public string legal_name;
        public bool? legal_name_needs_agreement;
        public string phone_number;
        public bool? phone_number_needs_agreement;
        public Profile profile;
        public bool? profile_needs_agreement;
    }
}

File: Android/app/src/main/java/com/rrrfffrrr/unity/kakaotalk/model/Account.kt
package com.rrrfffrrr.unity.kakaotalk.model

import com.kakao.sdk.user.model.Account
import com.kakao.sdk.user.model.AgeRange
import com.kakao.sdk.user.model.Gender
import com.kakao.sdk.user.model.Profile
import java.util.*

data class Account (
        val profileNeedsAgreement: Boolean?,
        val profile: Profile?,

        val emailNeedsAgreement: Boolean?,
        val isEmailValid: Boolean?,
        val isEmailVerified: Boolean?,
        val email: String?,

        val ageRangeNeedsAgreement: Boolean?,
        val ageRange: AgeRange?,

        val birthyearNeedsAgreement: Boolean?,
        val birthyear: String?,

        val birthdayNeedsAgreement: Boolean?,
        val birthday: String?,

        val genderNeedsAgreement: Boolean?,
        val gender: Gender?,

        val ciNeedsAgreement: Boolean?,
        val ci: String?,
        val ciAuthenticatedAt: Date?,

        val legalNameNeedsAgreement: Boolean?,
        val legalName: String?,

        val legalBirthDateNeedsAgreement: Boolean?,
        val legalBirthDate: String?,

        val legalGenderNeedsAgreement: Boolean?,
        val legalGender: Gender?,

        val phoneNumberNeedsAgreement: Boolean?,
        val phoneNumber: String?
    ) {
    constructor(account: Account) : this()
}